src/scss/_responsive-flat.scss

```diff
@@ -5,64 +5,65 @@
 
 /// Make a table 'ResponsiveFlat'
 @mixin oTableResponsiveFlat {
-	float: left;
-	width: 100%;
-
-	*,
-	*:before,
-	*:after {
-		box-sizing: border-box;
-	}
-
-	@include oGridRespondTo(S) {
-		display: table;
-	}
+	@include oBrandConfigureFor('o-table') {
+		float: left;
+		width: 100%;
 
-	@include oGridRespondTo($until: S) {
-		.o-table__cell--numeric {
-			text-align: left;
+		*,
+		*:before,
+		*:after {
+			box-sizing: border-box;
 		}
-	}
-
-	td {
-		padding: 8px;
-		width: 50%;
 
 		@include oGridRespondTo(S) {
-			border-right: 1px solid oColorsGetColorFor(o-table-row-right, border-right);
-			display: table-cell;
+			display: table;
 		}
-	} // td
-
-	thead th {
-		display: none;
-		padding: 8px;
 
-		@include oGridRespondTo(S) {
-			border-right: 1px solid oColorsGetColorFor(o-table-row-right, border-right);
-			display: table-cell;
+		@include oGridRespondTo($until: S) {
+			.o-table__cell--numeric {
+				text-align: left;
+			}
 		}
-	} // thead th
-
-	tr {
-		display: flex;
-		flex-flow: row wrap;
-		width: 100%;
 
-		@include oGridRespondTo(S) {
-			display: table-row;
-		}
-	} // tr
+		td {
+			padding: 8px;
+			width: 50%;
 
-	tbody tr th {
-		border-right: 1px solid oColorsGetColorFor(o-table-row-right, border-right);
-		float: left;
-		padding: 8px;
-		width: 50%;
+			@include oGridRespondTo(S) {
+				border-right: 1px solid oBrandGet('table-border-color');
+				display: table-cell;
+			}
+		} // td
 
-		@include oGridRespondTo(S) {
+		thead th {
 			display: none;
-		}
-	} // td th
+			padding: 8px;
 
+			@include oGridRespondTo(S) {
+				border-right: 1px solid oBrandGet('table-border-color');
+				display: table-cell;
+			}
+		} // thead th
+
+		tr {
+			display: flex;
+			flex-flow: row wrap;
+			width: 100%;
+
+			@include oGridRespondTo(S) {
+				display: table-row;
+			}
+		} // tr
+
+		tbody tr th {
+			border-right: 1px solid oBrandGet('table-border-color');
+			float: left;
+			padding: 8px;
+			width: 50%;
+
+			@include oGridRespondTo(S) {
+				display: none;
+			}
+		} // td th
```

Comment:
I thought these were accidental comments 🤔 I get that they're meant to indicate the end of a block, so I suppose this should be `tbody tr th`? Generally, though, it strikes me as a little odd if its not replicated across other files...Is there any reason for that?

Reply:
This was there before, it shows in the diff as I increased the indent. I agree it's useful but relies too much on developer discipline to use everywhere correctly. I decided not to remove but not to continue these kinds of comments.